browser-test/src/support/index.ts

```diff
@@ -189,10 +189,16 @@ export const createTestContext = (clearDb = true): TestContext => {
   // we'll get one huge video for all tests.
   async function resetContext() {
     if (browserContext != null) {
-      if (!DISABLE_BROWSER_ERROR_WATCHER) {
-        ctx.browserErrorWatcher.failIfContainsErrors()
+      try {
+        if (!DISABLE_BROWSER_ERROR_WATCHER) {
+          ctx.browserErrorWatcher.failIfContainsErrors()
+        }
+      } finally {
+        // browserErrorWatcher might throw error that should be bubble up all
+        // the way to user. But even if error is thrown we need to close the
+        // browser context, for example to save videos.
```

Comment:
Nit: some articles and a "d" are missing

browserErrorWatcher might throw an error that should be bubbled up all the way to the user. However even if an error is thrown...

Reply:
Thanks. Done.